### PR TITLE
Clustergrammer simplification

### PR DESCRIPTION
**Pull request by scvannost**
Simplified class _ClustergrammerTransform into a single function _clustergrammer_transform
Rewrote tests to accomodate

### CODE REVIEW

File: functions/visualizations.py [hunk 1]
```diff
@@ -98,76 +101,62 @@ def _ihc_combined_transform(
...
-        ):
-            return self.cytof_summary(data_file, metadata_df)
-
+    # if file_record.data_format.lower() == "npx":
+    #     return self.npx(data_file, metadata_df)
+    # elif file_record.upload_type.lower() in (
+    #     "cell counts compartment",
+    #     "cell counts assignment",
+    #     "cell counts profiling",
+    # ):
```
Comment (curlup) [inline functions/visualizations.py:116]: feels like overall needs a clean up

**Review by curlup — COMMENTED** [review]
any plans to push this forward?

**Review by curlup — APPROVED** [review]
looks good, though seems that this https://github.com/CIMAC-CIDC/cidc-cloud-functions/pull/162 was not completely taken out of here, so it's would be a merge conflict between those two I think.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-repos:
-  - repo: https://github.com/psf/black
-    rev: 19.3b0
-    hooks:
-      - id: black
+repos:
+  - repo: https://github.com/psf/black
+    rev: 19.3b0
+    hooks:
+      - id: black
```
Comment (curlup) [inline .pre-commit-config.yaml:5]: not sure why this one has changed?

Comment (scvannost) [reply .pre-commit-config.yaml:5]: no idea either
doesn't look like anything's changed

**Comment by curlup** [comment]
just curious why are you not merging ?

**Comment by scvannost** [comment]
> just curious why are you not merging? it is approved...

trying to deal with conflicts locally lead to a lot of things just breaking the build, so I need to fix that first

**Pull request closed without merging**